web/src/app/users/UserContactMethodEditDialog.js

```diff
@@ -25,57 +25,30 @@ const mutation = gql`
   }
 `
 
-export default class UserContactMethodEditDialog extends React.PureComponent {
-  static propTypes = {
-    contactMethodID: p.string.isRequired,
-    onClose: p.func,
-  }
-
-  state = {
-    value: null,
-    errors: [],
-    edit: true,
-  }
-
-  render() {
-    return (
-      <Query
-        query={query}
-        variables={{ id: this.props.contactMethodID }}
-        render={({ data }) => this.renderMutation(data.userContactMethod)}
-        noPoll
-      />
-    )
-  }
-
-  renderMutation({ name, type, value }) {
-    return (
-      <Mutation mutation={mutation} onCompleted={this.props.onClose}>
-        {(commit, status) =>
-          this.renderDialog(commit, status, { name, type, value })
-        }
-      </Mutation>
-    )
-  }
+export default function UserContactMethodEditDialog({
+  onClose,
+  contactMethodID,
+}) {
+  const [value, setValue] = useState(null)
+  const [edit, setEdit] = useState(true)
```

Comment:
```suggestion
  const [edit] = useState(true)
```

Reply:
It doesn't look like this is used anywhere other than passing as a prop, so we can probably just delete this line and do that directly below, on line 61:
```
edit={true}
```